docs/document/docs/guides/tooltip-and-field-group-customization.md

```diff
@@ -0,0 +1,269 @@
+```
+
+#### Add Event listener for FieldGroup 
+Add event listener for FieldGroup to change the tooltip message when FieldGroup is expanded or collapsed
+
+```javascript
+   fieldGroup.addEventListener('change', e => {
+    if (e.detail.expanded) {
+      tooltipForFieldGroup.title = '';
+    } else {
+      tooltipForFieldGroup.title = 'Office day info';
+    }
+  });
+```
+
+#### The display on UI
+![render](/img/tooltip_fieldgroup_customize.gif)
+
+
+### Using Two Types of Tooltip
+You may have noticed that when creating a Tooltip for a button, I will introduce a new property, describeChild. 
+This property is set to false by default.<br/>
+You can effectively improve the accessibility and usability of your application with the describeChild property.<br/>
+Now, let's enable the screen reader and try moving the focus to the button to see what the screen reader reads.
+```javascript
+  const tooltipForButton = new Kuc.Tooltip({
+    title: 'Submit changes or additions for this entry.',
+    container: button,
+    describeChild: false,
+    placement: 'bottom'
+  });
+```
+
+#### The display on UI
+![render](/img/tooltip_describeChild_false.gif)
+![render](/img/tooltip_describeChild_false.png)
+
+In the examples, with describeChild set to false, the screen reader replaces the button's original content 'Submit' with the Tooltip's content ('Submit changes or additions for this entry') when the button gains focus.<br/>
+This is because when describeChild is false, Tooltip adds [aria-label](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label) to its child elements (button, FieldGroup), causing the screen reader to read the content of [aria-label](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label) instead of the original content.
+
+When describeChild is set to true, let's see what the screen reader reads.
+```javascript
+  const tooltipForButton = new Kuc.Tooltip({
+    title: 'Submit changes or additions for this entry.',
+    container: button,
+    describeChild: true,
+    placement: 'bottom'
+  });
+```
+#### The display on UI
+![render](/img/tooltip_describeChild_true.gif)
+![render](/img/tooltip_describeChild_true.png)
+
+This time, we can see that when describeChild is set to true, the screen reader first reads the content of the button itself 'Submit' when the button gains focus, then continues to read the Tooltip's title 'Submit changes or additions for this entry.'.<br/>
+This is because when describeChild is true, Tooltip adds [aria-describedby](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-describedby) to its child elements (button), causing the screen reader to read the original content first and then continue with the content of [aria-describedby](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-describedby).
+
+
+:::info
+This article was reviewed by Kintone and Google Chrome as of Dec, 2023.<br/>
```

Comment:
This will be released in the next version, so how about changing it to Feb, 2024?